.circleci/Dockerfile

```diff
@@ -0,0 +1,11 @@
+# Add any extra build dependencies we need for running draupnir tests
+FROM circleci/golang:1.9
+
+RUN set -x \
+    && sudo apt-get update \
+    && sudo apt-get install -y \
+        build-essential \
+        ruby-dev \
+    && sudo gem install bundler fpm \
+    && sudo apt-get clean -y \
+    && sudo rm -rf /var/lib/apt/lists/*
```

Comment:
why this?

Reply:
Specifically the `rm`? because you want your container to be as small as possible, and this removes all the junk that apt downloads when you interact with the repositories (that you don't need to run your container)